src/Traits/LoadsFileTrait.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace Michaels\Manager\Traits;
+
+trait LoadsFileTrait
```

Comment:
I don't think `addFiles()` belongs here. All that really belongs here is a `load` method. See the end comment for more on that.